src/shared/Dropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, JSX, Show, createSignal } from "solid-js";
+
+import { ChevronDown } from "lucide-solid";
+import Button from "./Button";
+
+
+const Dropdown: Component<{ children: JSX.Element; label: string }> = (
```

Comment:
Any thoughts on a `DropdownItem` component to contain each `<a />` tag that goes into the `Dropdown`? Something like...

```jsx
<Dropdown>
  <DropdownItem href="#">Link Item label</DropdownItem>
  <DropdownItem onClick={() => doSomeThing()}>Button Item label</DropdownItem>
  ...
</Dropdown>
```

I think it would be a good idea to keep a standard component for what goes inside the `Dropdown`. I'm not too familiar with SolidJS, but it would be helpful to be able to optionally pass along things like prefix/suffix icons accessibility info, etc. while maintaining sane defaults.

I'm not terribly familiar with SolidJS; sorry if there's some concept I'm missing!

Reply:
Yep, I'm on board with this one! We could then drop the custom `dropdown-item` class and just apply the required classes directly inside the `DropdownItem` implementation, too.

Reply:
changed in latest commit